src/pages/usage.md

```diff
@@ -0,0 +1,101 @@
+* [Google Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Google+Refine%22+-openrefine&btnG=)
+* [Open Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Open+Refine%22+-openrefine&btnG=)
+* [OpenRefine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22OpenRefine%22&btnG=)
+
+
+```mermaid 
+%%{init: {'theme':'forest'}}%%
+xychart-beta
+    title "Number of academic citations of OpenRefine per years"
+    x-axis [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023]
+    y-axis "Count of citation" 0 --> 900
+    bar [4, 67, 162, 251, 356, 424, 477, 502, 584, 571, 586, 717, 783, 811]
+```
+
+## Forum Statistic 
+
+On November 2022, we moved from our Google Groups to discourse to host our forum. 
+
+As of March 20, 2024, over the last 12 months
+* 244 new users signed up on our forum for a total of 470 users. 
+* 324 topics were created for a total of 1,700 messages.
+
+## Contribution Statistics
+
+The statistics below only track activity on our primary GitHub repository, which includes code, design, and translation contributions. Our documentation is available in a separate repository.
+
+As of March 20th, 2024, over the last 12 months we have: 
+* 43 active github contributors (including dependabot and weblate);
+* [226 issues created](https://github.com/OpenRefine/OpenRefine/issues?q=is%3Aissue+created%3A%3E2023-03-13+);
+* [278 PR created](https://github.com/OpenRefine/OpenRefine/pulls?q=is%3Apr+created%3A%3E2023-03-13) (excluding PR created by [dependabot](https://github.com/OpenRefine/OpenRefine/pulls?q=is%3Apr+created%3A%3E2023-03-13+dependabot)) and [263 merged](https://github.com/OpenRefine/OpenRefine/pulls?q=is%3Amerged+created%3A%3E2023-03-13+);
+
+The following graphic represent the number of active contributors per months on the main repository [^1]. You can also review the [Github pulse](https://github.com/OpenRefine/OpenRefine/pulse) and [Github traffic](https://github.com/OpenRefine/OpenRefine/graphs/traffic) pages for real time insights on the activities on our main repository. 
+
+```mermaid 
+%%{init: {'theme':'forest'}}%%
+xychart-beta 
+    x-axis   2010 --> 2023
+    y-axis "Number of GitHub contributors per month" 0 --> 30 
+    line [1,2,3,5,4,3,2,2,4,5,4,4,2,3,0,2,3,2,2,3,2,3,3,2,2,1,2,1,1,1,2,2,2,1,3,4,2,1,1,4,2,3,1,2,0,2,2,3,0,2,5,4,3,2,6,4,0,4,1,8,3,5,0,1,0,0,0,1,1,0,1,2,1,2,0,0,5,5,2,4,4,6,8,3,12,9,12,12,17,13,14,14,6,8,8,9,10,16,12,8,4,10,7,4,4,9,6,8,11,7,4,13,11,27,25,15,22,21,18,18,16,19,7,19,23,14,22,16,12,7,9,12,15,18,12,18,17,19,21,11,14,17,13,7,14,14,10,9,10,13,13,12,10,8,7,9,9,8,6,12]
+```
+
```

Comment:
This very noisy. Does the graph package support adding something like a 3-month rolling average for smoothing?

Reply:
We will need to compute the 3-month rolling average and update the x and y axes accordingly. Mermaid does not have computing capabilities.

Reply:
I've computed the rolling average and regenerated the chart.